aap-common/assembly-aap-command-generator.adoc

```diff
@@ -0,0 +1,43 @@
+ifdef::context[:parent-context: {context}]
+
+[id='aap-command-generator']
+
+
+= Using the Command Generator
+
+The Command Generator is used to generate commands for launching operational playbooks provided by the Ansible-on-clouds operational playbook collection.
+
+The process involves five steps:
+
+. Pull the `ansible-on-clouds-ops`` container image.
+. List the available playbooks.
+. Use a command generator to generate a data file and the next command to run.
+. Populate the data file and launching the previous generated command.
+This generates the final command with all parameters.
++
+[NOTE]
+====
+When this step is complete, you can save the generated command and run the playbook when it is required.
+====
++
+. Running the final command.
+
+`command_generator_vars` and the command_generator are implemented using a docker container and are launched using the docker command line interface.
+
+The docker image is available at:
+
+.Prerequisites
+* Docker
+ifdef::product_AWS[]
+* An AWS credentials file
+endif::product_AWS[]
+ifdef::product_GCP[]
+* A GCP credentials file
+endif::product_GCP[]
+* An internet connection to Google Cloud
```

Comment:
Should this be located in a separate section?
Asking because the location isn't given for the docker image, and docker is mentioned before the "docker" prerequisite
Or is line 25 part of the Running the final command step (#5) ?

Reply:
![image](https://github.com/ansible/aap-docs/assets/44700011/89cecc89-2a21-403f-a446-413163f70967)